src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: '0',
+    };
+  }
+
+  render() {
+    const { result } = this.state;
+    return (
+      <div>
+        <Display result={result} />
+        <ButtonPanel />
+      </div>
+    );
```

Comment:
- [ ] To avoid linter error `App` should be a functional component ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/appFunctionalComponent.png)). Kindly fix it.